[DOCS] Add Users page to Security UI (#1931)
elastic/security-docs: docs/getting-started/security-ui.asciidoc

* Host isolation exceptions
* Blocklist

Pages are grouped into four main sections within the navigation pane -- *Detect*, *Explore*, *Investigate*, and *Manage*. Each section supports a different part of your workflow and provides actions you can perform in the {security-app}.

From the *Detect* section, access pages that allow you to view, create, and manage alerts, rules, and rule exceptions.

Under *Explore*, access pages that provide key metrics about your hosts and networks.

The *Investigate* section gives you access to Timelines and cases.

> Pages under the *Manage* section allow you to view and manage hosts that are running Endpoint Security. You can also view and manage trusted applications and event filters from pages in this section.

Reviewer: Even if we keep these as full sentences, I'd still omit the last sentence here -- it's outdated and I don't think we need to list all the Endpoint artifacts here, just mention that *Manage* is related to Endpoint Security.
```suggestion
Pages under the *Manage* section allow you to view and manage hosts that are running {endpoint-sec}.
```

From the live feed on the *Overview* page, you can quickly access recently created cases, favorited timelines, and the latest {elastic-sec} news.

TIP: The *Security news* section provides you with the latest {elastic-sec} news to stay informed on new developments, learn about {elastic-sec} features, and more.

image::images/live-feed-ov-page.png[Shows the Overview page]

*Histograms*

> Time-based histograms show you the number of detections, alerts, and events that have occurred within the selected time range. You can select areas of interest in time-based histograms by selecting a region to reflect a date range, or selecting a preset value in the timepicker. In the **Stack by** dropdown, you can select specific parameters to visualize individual counts. For example, in the Detection alert trend histogram, stack by `kibana.alert.rule.name` to display the total counts by alert name within the specified time frame.

Reviewer: Possible revision for a little more conciseness and verb variety.
```suggestion
Time-based histograms show you the number of detections, alerts, and events that have occurred within the selected time range. To focus on areas of interest in time-based histograms, select a region to reflect a date range, or select a preset value in the timepicker. In the **Stack by** dropdown, you can select specific parameters to visualize individual counts. For example, in the Detection alert trend histogram, stack by `kibana.alert.rule.name` to display the total counts by alert name within the specified time frame.
```

[[hosts-ui]]
=== Hosts page

> The Hosts page provides key metrics for host-related security events and a set of interactive data tables. Refer to <<hosts-overview, Hosts page overview>> for more information.

Reviewer 2: ```suggestion
The Hosts page provides key metrics for host-related security events and a set of interactive data tables. Refer to <<hosts-overview, Hosts page>> for more information.
```
